mrchecker-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/mrchecker/database/core/BaseDatabase.java

```diff
@@ -0,0 +1,124 @@
+package com.capgemini.mrchecker.database.core;
+
+import com.capgemini.mrchecker.database.core.base.properties.PropertiesFileSettings;
+import com.capgemini.mrchecker.database.core.base.runtime.RuntimeParameters;
+import com.capgemini.mrchecker.test.core.BaseTest;
+import com.capgemini.mrchecker.test.core.BaseTestWatcher;
+import com.capgemini.mrchecker.test.core.ITestObserver;
+import com.capgemini.mrchecker.test.core.ModuleType;
+import com.capgemini.mrchecker.test.core.analytics.IAnalytics;
+import com.capgemini.mrchecker.test.core.base.environment.IEnvironmentService;
+import com.capgemini.mrchecker.test.core.base.properties.PropertiesSettingsModule;
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.google.inject.Guice;
+import lombok.Getter;
+
+import javax.persistence.EntityManager;
+
+abstract public class BaseDatabase implements ITestObserver, IDatabasePrefixHolder {
```

Comment:
Please we keep naming convention 
instead "BaseDatabase", could be "BasePageDatabase" ?